Pack18/Task01/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Даны 2 числа и операция. Вывести на консоль выбранный  байт ответа. 
```

Comment:
Now I finally don't understand anything. We have binary tasks here and in the new pull request and also we completed the PR with these tasks...